config/config.yaml

```diff
@@ -192,3 +192,9 @@ tls:
   #
   # Required if cluster.p2p.enable_tls is true.
   ca_cert: ./tls/cacert.pem
+
+  # TTL to re-load certificate from disk. Useful for certificate rotations,
+  # Only works for HTTPS endpoints, gRPC endpoints (including intra-cluster communication)
+  # doesn't support certificate re-load
+  cert_ttl: 3600
```

Comment:
This should mention it is in seconds.

Also, this value is different than

```rust
// Default one
60 * 60 * 24
```

in `settings.rs`. I think we should pick one.

Reply:
Thanks, I picked 1 hour here as the default.